scripts/cleansing.py

```diff
@@ -3,23 +3,30 @@
 from typing import Callable
 
 
+def change_float_to_int(dataset: DataFrame, columns : list):
+    dataset[columns] = dataset[columns].astype('int')
+
+
 def read_and_cleanse(dataset_path : str,
```

Comment:
maybe it's better to separate reading from cleaning and just call two functions in main